operators/src/processing/meteosat/radiance.rs

```diff
@@ -0,0 +1,502 @@
+use crate::engine::{
+    ExecutionContext, InitializedRasterOperator, Operator, QueryContext, QueryProcessor,
+    RasterOperator, RasterQueryProcessor, RasterQueryRectangle, RasterResultDescriptor,
+    SingleRasterSource, TypedRasterQueryProcessor,
+};
+use crate::util::Result;
+use futures::stream::BoxStream;
+use futures::StreamExt;
+use geoengine_datatypes::primitives::SpatialPartition2D;
+use geoengine_datatypes::raster::{
+    EmptyGrid, Grid2D, GridShapeAccess, Pixel, RasterDataType, RasterPropertiesKey, RasterTile2D,
+};
+use num_traits::AsPrimitive;
+use std::convert::TryFrom;
+
+use async_trait::async_trait;
+use serde::{Deserialize, Serialize};
+
+//TODO Check
+type PixelOut = f32;
+use RasterDataType::F32 as RasterOut;
+use TypedRasterQueryProcessor::F32 as QueryProcessorOut;
+
+//TODO: For NAN the tests fail due to == comparisons
+const OUT_NO_DATA_VALUE: PixelOut = PixelOut::NEG_INFINITY;
```

Comment:
This is a good point. For our collections, we have an equality method that treats NAN = NAN.
Maybe we can create a util method to check rasters with this semantics as well for tests.

Reply:
I implemented a utility method for this problem here: bdc01c5bcb17c2432b7607672a9f25f5042eeaec
And changed the affected tests here: 047d20b50f49a80763ecbe99f6533302675caadc